src/test/java/io/personium/core/model/CellTest.java

```diff
@@ -0,0 +1,140 @@
+package io.personium.core.model;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.personium.common.auth.token.IExtRoleContainingToken;
+import io.personium.common.auth.token.Role;
+import io.personium.common.auth.token.TransCellAccessToken;
+import io.personium.common.utils.CommonUtils;
+import io.personium.core.PersoniumUnitConfig;
+import io.personium.core.utils.UriUtils;
+
+public class CellTest {
+    static Cell testCell;
+    static final String CELL_NAME = "testcell";
+    static final String CELL_ID = "12345";
+    static final String CELL_OWNER = "somebody";
+    static Logger log = LoggerFactory.getLogger(CellTest.class);
+    static volatile String fqdnShelter;
+
+    @BeforeClass
+    public static void setUpBeforeClass() throws Exception {
+        fqdnShelter = CommonUtils.getFQDN();
+        TransCellAccessToken.configureX509(PersoniumUnitConfig.getX509PrivateKey(),
+                PersoniumUnitConfig.getX509Certificate(), PersoniumUnitConfig.getX509RootCertificate());
+
+        CommonUtils.setFQDN("testunit.example");
+        PersoniumUnitConfig.set(PersoniumUnitConfig.UNIT_PORT, "");
+        PersoniumUnitConfig.set(PersoniumUnitConfig.UNIT_SCHEME, "https");
+
+        testCell = new Cell() {
+            @Override
+            public String getDataBundleNameWithOutPrefix() {
+                return null;
+            }
+
+            @Override
+            public String getDataBundleName() {
+                return null;
+            }
+
+            @Override
+            public void makeEmpty() {
+
+            }
+            @Override
+            public void delete(boolean recursive, String unitUserName) {
+
+            }
+
+            @Override
+            public List<Role> getRoleListForAccount(String username) {
+                return null;
+            }
+
+            @Override
+            public List<Role> getRoleListHere(IExtRoleContainingToken token) {
+                return null;
+            }
+            @Override
+            public String roleIdToRoleResourceUrl(String roleId) {
+                return null;
+            }
+            @Override
+            public String roleResourceUrlToId(String roleUrl, String baseUrl) {
+                return null;
+            }
+
+        };
+        testCell.id = CELL_ID;
+        testCell.name = CELL_NAME;
+        testCell.owner = CELL_NAME;
```

Comment:
```suggestion
        testCell.owner = CELL_OWNER;
```

Or remove this line since testCell.owner is always updated on testcode.